docs/advanced/stateful-processing.md

```diff
@@ -158,3 +158,9 @@ for the state to become slightly out of sync with a topic in between shutdowns a
 While the impact of this is generally minimal and only for a small amount of messages, be aware this could cause side effects where the same message may be reprocessed differently, if it depended on certain state conditionals.
 
 "Exactly Once" delivery guarantees avoid this. You can learn more about delivery/processing guarantees [here](https://quix.io/docs/quix-streams/configuration.html?h=#processing-guarantees).
+
+## Serialization
+
+By default, the keys and values are serialized to JSON for storage. If you need to change the serialization format, you can do so using the `rocksdb_options` parameter when creating the `Application` object. This change will apply to all state stores created by the application and existing state will be un-readable.
```

Comment:
If we mention it, then lets explain how exactly user can use `rocksdb_options` for it. Maybe en example snippet?

Reply:
done